web/src/settings_playgrounds.ts

```diff
@@ -149,28 +159,29 @@ function build_page() {
             });
         });
 
-    const $search_pygments_box = $("#playground_pygments_language");
-    let language_labels = new Map();
+    const $search_pygments_box = $<HTMLInputElement>("input#playground_pygments_language");
```

Comment:
Can you double-check if `playground_pygments_language` is inside an input tag. I found this

```
<td>
        <span class="playground_pygments_language">{{pygments_language}}</span>
    </td>
```
inside `admin_playground_list.hbs`. 
```
("span#playground_pygments_language")
```
might be correct here ? Would be great if you can clarify.

Reply:
Nope! it should be `input#playground_pygments_language` since `#` is a selector which select the id not the class, for class we use `.` 
For more clarity check `playground_settings_admin.hbs` we have an input element in line 47 with id `playground_pygments_language`